app/_mesh_redirects

```diff
@@ -0,0 +1,21 @@
+# TODO: Automate this file's generation
+# Generated redirects
+/mesh/:version/overview/ /mesh/:version/overview/what-is-a-service-mesh 301
```

Comment:
```suggestion
/mesh/:version/overview/ /mesh/:version/introduction/what-is-a-service-mesh 301
```
Tested, this is the topic: https://deploy-preview-5079--kongdocs.netlify.app/mesh/latest/introduction/what-is-a-service-mesh/